sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSql.java

```diff
@@ -25,70 +25,68 @@
 import org.apache.beam.sdk.values.TupleTag;
 
 /**
- * {@code BeamSql} is the DSL interface of BeamSQL. It translates a SQL query as a
- * {@link PTransform}, so developers can use standard SQL queries in a Beam pipeline.
+ * {@code BeamSql} is the DSL interface of BeamSQL. It translates a SQL query as a {@link
+ * PTransform}, so developers can use standard SQL queries in a Beam pipeline.
  *
  * <h1>Beam SQL DSL usage:</h1>
- * A typical pipeline with Beam SQL DSL is:
- * <pre>
- *{@code
-PipelineOptions options = PipelineOptionsFactory.create();
-Pipeline p = Pipeline.create(options);
-
-//create table from TextIO;
-PCollection<Row> inputTableA = p.apply(TextIO.read().from("/my/input/patha")).apply(...);
-PCollection<Row> inputTableB = p.apply(TextIO.read().from("/my/input/pathb")).apply(...);
-
-//run a simple query, and register the output as a table in BeamSql;
-String sql1 = "select MY_FUNC(c1), c2 from PCOLLECTION";
-PCollection<Row> outputTableA = inputTableA.apply(
-    BeamSql
-        .query(sql1)
-        .registerUdf("MY_FUNC", MY_FUNC.class, "FUNC");
-
-//run a JOIN with one table from TextIO, and one table from another query
-PCollection<Row> outputTableB =
-    PCollectionTuple
-        .of(new TupleTag<>("TABLE_O_A"), outputTableA)
-        .and(new TupleTag<>("TABLE_B"), inputTableB)
-        .apply(BeamSql.query("select * from TABLE_O_A JOIN TABLE_B where ..."));
-
-//output the final result with TextIO
-outputTableB.apply(...).apply(TextIO.write().to("/my/output/path"));
-
-p.run().waitUntilFinish();
- * }
- * </pre>
  *
+ * <p>A typical pipeline with Beam SQL DSL is:
+ *
+ * <pre>{@code
```

Comment:
You might want to put this into a block quote so the custom formatting for pre isn't messed up.